support/crown/src/common.rs

```diff
@@ -49,22 +48,22 @@ pub fn in_derive_expn(span: Span) -> bool {
 
 #[macro_export]
 macro_rules! symbols {
-    ($($s: ident)+) => {
-        #[derive(Clone)]
-        #[allow(non_snake_case)]
-        pub(crate) struct Symbols {
-            $( $s: Symbol, )+
-        }
-
-        impl Symbols {
-            fn new() -> Self {
-                Symbols {
-                    $( $s: Symbol::intern(stringify!($s)), )+
-                }
-            }
-        }
-    }
-}
+     ($($s: ident)+) => {
+         #[derive(Clone)]
+         #[allow(non_snake_case)]
+         pub(crate) struct Symbols {
+             $( $s: Symbol, )+
+         }
+
+         impl Symbols {
+             fn new() -> Self {
+                 Symbols {
+                     $( $s: Symbol::intern(stringify!($s)), )+
+                 }
+             }
+         }
+     }
+ }
```

Comment:
This is accident I assume (normal formatting has 4 spaces not 5).

Reply:
Sorry, There are not intended. Thank you for your advice.